src/systempolicy/systemPolicy.go

```diff
@@ -769,6 +769,8 @@ func GenFileSetForAllPodsInCluster(clusterName string, pods []types.Pod, settype
 		}
 		dbfs := out[wpfs]
 		if len(dbfs) == 0 {
+			wpfs.CreatedTime = uint(time.Now().Unix())
```

Comment:
CreatedTime/UpdateTime should be handled internally in the mysqlhandler functions.